content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,11 +72,19 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/total
+
+Example: `apiserver.latency.k8s.io/total: "573.018706ms"`
+
+This value shows total latency incurred in components within the apiserver. Please note that the latency is recorded in
+annotation only when the total latency of the given request exceeds 500ms; This is in keeping with the
+traces in rest/handlers for create, delete, update, get, list, and delete collection.
```

Comment:
```suggestion
This annotation records total latency incurred in components within the API server.
The API server only adds this audit annotation for requests where the total latency for handling
and responding to that request exceeds a threshold; in Kubernetes {{< skew currentVersion >}}
that threshold is 500ms (half a second).
```

Reply:
This looks clean , thanks for the suggestion.